spec/dummy/config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  mount ScimRails::Engine => "/scim_rails"
```

Comment:
All of the examples in the readme assume that the engine is mounted at the base path. This changes the dummy app to make that true so that the examples in the readme work.